paasta_tools/cli/cmds/spark_run.py

```diff
@@ -257,6 +257,14 @@ def add_subparser(subparsers):
         default="default",
     )
 
+    aws_group.add_argument(
+        "--no-aws-credentials",
+        help="Do not load any AWS credentials; allow the service to use its "
```

Comment:
ultra-mega-nits: it's really the spark job that is doing this, right?  the service is sort of a more broad term.  Also you have the word "its" there twice